client/src/Components/Suggestions.js

```diff
@@ -4,25 +4,75 @@ import List from "@material-ui/core/List";
 import ListItem, { ListItemProps } from "@material-ui/core/ListItem";
 import ListItemIcon from "@material-ui/core/ListItemIcon";
 import ListItemText from "@material-ui/core/ListItemText";
+import { withStyles } from "@material-ui/core/styles";
+import SearchIcon from "@material-ui/icons/Search";
+
+const styles = theme => ({
+  searchList: {
+    display: "flex",
+    flexDirection: "column",
+    backgroundImage: "linear-gradient(white, #e0eff9)",
+    color: "black",
+    fontFamily: "Roboto"
+  },
+  header: {
+    paddingLeft: "1em"
+  },
+  listItem: {
+    flex: 1,
+    fontWeight: 350
+  },
+  queryResult: {
+    display: "flex",
+    justifyContent: "space-evenly",
+    alignItems: "center"
+  },
+  queryLabel: {
+    flex: 4
+  },
+  searchIcon: {
+    width: theme.spacing.unit * 5,
+    marginRight: "0.5em",
+    pointerEvents: "none",
+    flex: 0.1
+  }
+});
 
 class Suggestions extends Component {
   handleClick = (e, article) => {
     this.props.handleSearchClick(e, article);
   };
 
   render() {
+    const { classes } = this.props;
     if (this.props.queryResultArticles !== undefined) {
-      const options = this.props.queryResultArticles.map(article => (
-        <ListItem
-          key={article.url}
-          button
-          onClick={e => this.handleClick(e, article)}
-        >
-          <ListItemText primary={article.title} />
-        </ListItem>
-      ));
+      const options = this.props.queryResultArticles.map(article => {
+        const text = {
+          __html: article.title
+            .toLowerCase()
+            .split(this.props.query.toLowerCase())
+            .join(`<b>${this.props.query.toLowerCase()}</b>`)
+        };
+        return (
+          <div className={classes.queryResult}>
```

Comment:
@mealeark There is new noise in the console around the div not having a key. Can you clean that up in a new PR when you have a minute?

Warning: Each child in a list should have a unique "key" prop.

Check the render method of `Suggestions`. See https://fb.me/react-warning-keys for more information.
    in div (at Suggestions.js:57)
    in Suggestions (created by WithStyles(Suggestions))
    in WithStyles(Suggestions) (at LandingPage.js:128)
    in div (created by Grid)
    in Grid (created by WithStyles(Grid))
    in WithStyles(Grid) (at LandingPage.js:121)
    in div (created by Grid)
    in Grid (created by WithStyles(Grid))
    in WithStyles(Grid) (at LandingPage.js:120)
    in div (created by Paper)
    in Paper (created by WithStyles(Paper))
    in WithStyles(Paper) (at LandingPage.js:119)
    in div (at LandingPage.js:118)
    in LandingPage (created by WithStyles(LandingPage))
    in WithStyles(LandingPage) (at App.js:8)
    in div (at App.js:7)
    in App (at src/index.js:6)